### ISSUE TITLE
Not reading input 

### ISSUE THREAD

**Issue by wodinga** [mcollina/autocannon#136]
When I run autocannon from the CLI and specify an input file it doesn’t work. 

I’ve also tried using the API in a node app and my app still reports an empty body

**Comment by mcollina** [comment]
Can you please provide an example to reproduce along with a server example?

**Comment by davidmarkclements** [comment]
@wodinga @zerodom30 - in order to assess, a server example is needed

**Comment by zerodom30** [comment]
@davidmarkclements @mcollina here is mock server: 
```
const express = require('express')
const bodyParser = require('body-parser')
const morgan = require('morgan')

const app = express()
const port = 5555
const env = process.env.NODE_ENV || 'local'
console.log('Setting env ', env);

app.use(bodyParser.urlencoded({ extended: true }));
app.use(bodyParser.json());
app.use(morgan("dev"));

app.post('/content', (req, res, next) => {
    console.log('Req:', req.body)
    res.json({ content: req.body })
})

app.listen(port, function () {
    console.log( 'Running Express server')
});
```

`autocannon -c 10 -m 'POST' -i 'body.json' localhost:5555/content`

body.json contents:
{ "contents": "1234567" }

This can be contrast with results of a `CURL` post.

**Comment by mcollina** [comment]
You are not specifying the content-type to parse that body:

```
autocannon -c 10 -m 'POST' -i 'body.json' -H 'Content-Type: application/json' localhost:5555/content
```

^ This works.

**Comment by mcollina** [comment]
@zerodom30 would you like to send a PR to address this documentation problem?

**Comment by amorriscode** [comment]
This PR was closed by #149. 👍
